src/Service/UserService.js

```diff
@@ -0,0 +1,18 @@
+export default class UserService {}
+
+UserService.login = (token, userName) => {
+  localStorage.setItem('token', token);
+  localStorage.setItem('userName', userName);
+};
+UserService.logout = () => {
+  localStorage.removeItem('token');
+  localStorage.removeItem('userName');
+};
+
+UserService.getUserName = () => {
+  return localStorage.getItem('userName');
+};
+
+UserService.isLoggedIn = () => {
```

Comment:
method which name started with 'is' should return boolean value
if you need method to return a token you should create separate method and name it accordingly